.spelling

```diff
@@ -200,6 +200,7 @@ Cale
 CDNs
 CentOS
 Cernich
+cert
```

Comment:
I don't believe cert is a valid word, although it may be an abbreviation. Remove this here, and update cert to certificate below.

Reply:
At the same time we do have cert in other areas so I don't think there was any error saying cert is actually misspelled and you might be able to just leave them.